src/main/java/com/dezorganizacja/bridge/presenter/MainPresenter.java

```diff
@@ -2,8 +2,11 @@
 
 import com.dezorganizacja.bridge.domain.Message;
 
-public class MainPresenter {
+import java.util.Observable;
+
+public class MainPresenter extends Observable {
     private final Message message;
+    private String state;
```

Comment:
Chyba nie chcemy by to był String. Jeśli mamy skończoną ilość elementów i ogólnie one są jakoś sparowane czyli tak jak w tym przypadku "main": "fxml/main.fxml", "loading": "fxml/loading.fxml" i tak dalej warto użyć enumów.